easybuild/easyconfigs/h/HPL/HPL-2.2-fosscuda-2018b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'HPL'
+version = '2.2'
+
+homepage = 'http://www.netlib.org/benchmark/hpl/'
+description = """HPL is a software package that solves a (random) dense linear system in double precision (64 bits)
+ arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available
+ implementation of the High Performance Computing Linpack Benchmark."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.netlib.org/benchmark/%(namelower)s']
+checksums = ['ac7534163a09e21a5fa763e4e16dfc119bc84043f6e6a807aba666518f8df440']
```

Comment:
missing checksum for patch, cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/6424/files#diff-21f46ca04de12d4b01d62a39ce2a0686R14